lib/Serialization/Deserialization.cpp

```diff
@@ -4713,17 +4715,20 @@ class swift::TypeDeserializer {
                                                   noescape,
                                                   // SWIFT_ENABLE_TENSORFLOW
                                                   throws,
-                                                  differentiable);
+                                                  rawDiffKind);
+      diffKind = DifferentiabilityKind(rawDiffKind);
+      
     } else {
       GenericSignatureID rawGenericSig;
       decls_block::GenericFunctionTypeLayout::readRecord(scratch,
                                                          resultID,
                                                          rawRepresentation,
                                                          throws,
                                                          // SWIFT_ENABLE_TENSORFLOW
-                                                         differentiable,
+                                                         rawDiffKind,
                                                          rawGenericSig);
       genericSig = MF.getGenericSignature(rawGenericSig);
+      diffKind = DifferentiabilityKind(rawDiffKind);
```

Comment:
```suggestion
      // SWIFT_ENABLE_TENSORFLOW
      diffKind = DifferentiabilityKind(rawDiffKind);
```

Reply:
Should I put one on [line 4719](https://github.com/apple/swift/pull/25325/files/c3b6c8133bf2e8f8d06d9c3e7ae0077469c42ec5#diff-5fa18b91a026a0b29b5c3b4d59feaf95R4719) in `Deserialzation.cpp` (this same file) as well?

Reply:
That place already has one.

Reply:
Resolved offline: I flipped the order of `genericSig` and `diffKind` so I could remove the flag.